apcd-cms/src/apps/registrations/templates/submission_form/registration_form_scripts.html

```diff
@@ -58,9 +58,17 @@
         {% else %}
             1
         {% endif %};
+      let reg_id = 
+        {% if r %}
+          Number(`
+            {{r.reg_id}}
```

Comment:
Can you combine 63-65 into one line? It seems a little more readable to me, but if it's just splitting hairs ignore me.